transformations/sentence_reordering/transformation.py

```diff
@@ -0,0 +1,61 @@
+import random
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+# for sent tokenizer
+import spacy
+
+nlp = spacy.load("en_core_web_sm")
+
+# coref resolution from allennlp
+# ref: https://demo.allennlp.org/coreference-resolution
+import allennlp_models.tagging
+from allennlp.predictors.predictor import Predictor
+
+predictor = Predictor.from_path(
+    "https://storage.googleapis.com/allennlp-public-models/coref-spanbert-large-2021.03.10.tar.gz"
```

Comment:
This needs to be removed since it will load twice now.

Reply:
oops somehow I didn't push the reformatted version. Now it should be fixed. Thanks for catching this!